From 96369919e5100865e2469e42fb8f4b8e38e41aef Mon Sep 17 00:00:00 2001
From: David Henningsson <david.henningsson@canonical.com>
Date: Mon, 4 Jul 2011 10:12:17 +0200
Subject: [PATCH 102/111] alsa-mixer: Set "Front" control to 0 dB on headphone
 path

I've seen more than one system where the volume control named
"Front" is a part of audio path for headphones. This is somewhat
of a compromise: While we don't merge it into the path, as that
would be regressing machines where "Front" isn't a part of the
audio path, it would still enable sound on these machines.

BugLink: http://bugs.launchpad.net/bugs/804178
Signed-off-by: David Henningsson <david.henningsson@canonical.com>
---
 .../mixer/paths/analog-output-headphones-2.conf    |    5 +++--
 .../alsa/mixer/paths/analog-output-headphones.conf |    5 +++--
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf b/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf
index e47543f..284713b 100644
--- a/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf
+++ b/src/modules/alsa/mixer/paths/analog-output-headphones-2.conf
@@ -60,9 +60,10 @@ volume = off
 switch = off
 volume = off
 
+; On some machines Front is actually a part of the Headphone path
 [Element Front]
-switch = off
-volume = off
+switch = mute
+volume = zero
 
 [Element Rear]
 switch = off
diff --git a/src/modules/alsa/mixer/paths/analog-output-headphones.conf b/src/modules/alsa/mixer/paths/analog-output-headphones.conf
index 1d8d3fe..7f95f0a 100644
--- a/src/modules/alsa/mixer/paths/analog-output-headphones.conf
+++ b/src/modules/alsa/mixer/paths/analog-output-headphones.conf
@@ -63,9 +63,10 @@ volume = off
 switch = off
 volume = off
 
+; On some machines Front is actually a part of the Headphone path
 [Element Front]
-switch = off
-volume = off
+switch = mute
+volume = zero
 
 [Element Rear]
 switch = off
-- 
1.7.5.4

